finality-aleph/src/party/mod.rs

```diff
@@ -160,18 +120,20 @@ where
     let session_authorities = Arc::new(Mutex::new(HashMap::new()));
     let block_requester = network.clone();
 
-    let verifier_timeout = Duration::from_millis(VERIFIER_TIMEOUT_MILISECONDS);
-    let notification_timeout = Duration::from_millis(NOTIFICATION_TIMEOUT_MILISECONDS);
+    let justification_handler_config: JustificationHandlerConfig<_> = Default::default();
 
     let handler = JustificationHandler::new(
         get_session_info_provider(session_authorities.clone(), session_period),
         block_requester.clone(),
         client.clone(),
         AlephFinalizer::new(client.clone()),
-        JustificationRequestDelayImpl::new(&session_period, &millisecs_per_block),
+        JustificationRequestSchedulerImpl::new(
+            &session_period,
+            &millisecs_per_block,
+            justification_handler_config.max_attemps(),
```

Comment:
Uhh, this is really strange. It might be better to construct this struct internally and pass only the config? This will be strange with testing though... maybe don't make `max_attempts` part of the config?

Reply:
I move it, but now there is a random constant in `party/mod.rs`. Neither of them is good for me, but I don't know how to make it better :disappointed: